index.html

```diff
@@ -49,5 +49,167 @@ <h1><span class="hero__title-dark">Hey there. I'm Daniel Selase</span><span clas
+            <a href="#" class="card__tags-links">css</a>
+            <a href="#" class="card__tags-links">JavaScript</a>
+            <a href="#" class="card__tags-links">html</a>
+        </div>
+        <button class="btn">See Project</button>
+</div>
+<div class="card">
+    <div class="card__img"></div>
+    <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
+    <div class="card__tags">
+        <a href="#" class="card__tags-links">Ruby on rails</a>
+        <a href="#" class="card__tags-links">css</a>
+        <a href="#" class="card__tags-links">JavaScript</a>
+        <a href="#" class="card__tags-links">html</a>
+    </div>
+    <button class="btn">See Project</button>
+</div>
+            </div>
+        </div>
+    </section>
+
+    <!-- About me -->
+    <section class="about">
+        <div class="container">
+            <h2 class="heading-2">About me</h2>
+        <p class="about__text">I can help you build a product, feature or website. Look through some of my work and experience. If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <button class="btn">Get My Resume</button>
+        </div>
+    </section>
+
+    <!-- Skills -->
+    <section class="skills">
+        <div class="container">
+            <div class="card">
+                <img src="./images/icon-languages.svg" alt="">
+                <h4 class="card__title">Languages</h4>
+                <div class="card__tags">
+                    <a href="#" class="card__tags-links">JavaScript</a>
+                    <a href="#" class="card__tags-links">Ruby</a>
+                    <a href="#" class="card__tags-links">Html</a>
+                    <a href="#" class="card__tags-links">CSS</a>
+                </div>
+            </div>
+            <div class="card">
+                <img src="./images/icon-frameworks.svg" alt="">
+                <h4 class="card__title">Frameworks</h4>
+                <div class="card__tags">
+                    <a href="#" class="card__tags-links">Bootstrap</a>
+                    <a href="#" class="card__tags-links">Ruby on fails</a>
+                    <a href="#" class="card__tags-links">Rspec</a>
+                    <a href="#" class="card__tags-links">Capybara</a>
+                    <a href="#" class="card__tags-links">Selenium</a>
+                </div>
+            </div>
+            <div class="card">
+                <img src="./images/icon-skills.svg" alt="">
+                <h4 class="card__title">Skills</h4>
+                <div class="card__tags">
+                    <a href="#" class="card__tags-links">Codekit</a>
+                    <a href="#" class="card__tags-links">Github</a>
+                    <a href="#" class="card__tags-links">Codepen</a>
+                    <a href="#" class="card__tags-links">Gitlab</a>
+                    <a href="#" class="card__tags-links">Terminal</a>
+                </div>
+            </div>
```

Comment:
- [ ]  I think it would be better to use an unordered list to wrap using `ul` and `li` instead of div for a semantic reason.